src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -99,14 +121,25 @@ const RunDataset = ({
 function buildDatasetDataMarkup(
   datasetKey,
   datasetValues,
+  datasetType,
   rowIndex,
   isSingleRun,
   pinnedRun,
   enableShowChanges,
-  selectedRunIds
+  selectedRunIds,
+  setRunDatasetToShow,
+  setShowRunVizModal
```

Comment:
This `buildDatasetDataMarkup` is a pretty hefty function. Would be nice if you could add your new arguments to the comment.